Makefile.standard_app

```diff
@@ -60,6 +60,14 @@ else
         DEFINES += PRINTF\(...\)=
 endif
 
+#####################################################################
+#                          LEDGER_ASSERT                            #
+#####################################################################
```

Comment:
Maybe when DEBUG is on, we should enable a bigger level of displayed info? Like the file and line?

Reply:
I was wondering the same thing Letsgo :)